### ISSUE TITLE
Alt-F-S doesn't save the current file

### ISSUE THREAD

**Issue by Fozzy0007** [Microsoft/vscode#5556]
- VSCode Version:
- OS Version:

Steps to Reproduce:
1. edit a file (VSCode marks it as needing to be saved)
2. Alt+F+S

file doesn't save.  in any other windows program, chording Alt+F (let go of F, keep holding Alt) + S will save the file.  

if you let go of "Alt" between F and S, it works.  my muscle memory is messing with me.

**Comment by Tyriar** [comment]
This is an Electron issue https://github.com/atom/atom/issues/10732

**Comment by Treit** [comment]
The discussion in issue atom/atom#10732 concludes that this is something different and _not_ the same issue as atom/atom#10732.

**Comment by Tyriar** [comment]
As mentioned in the atom issue, the underlying issue is related to electron, not vscode.

**Comment by Treit** [comment]
Is there an active Electron issue tracking this? I couldn't find one. I read through the issues described here (https://github.com/electron/electron/issues/2504) but none of them quite seem to match the behavior this issue is talking about, which is that a three-character combination like Alt+F+S does not do anything and you have to release the Alt key to make it work.

**Comment by Tyriar** [comment]
That does look like the upstream issue as it tracks a bunch of problems with it.

**Comment by SteveALee** [comment]
It's seems to be specific to ALT+F, S as other mnemonics in the file menu like A (Save AS)  P (prefs sub menu) work fine

**Comment by myfairsyer** [comment]
> other mnemonics in the file menu like A (Save AS) P (prefs sub menu) work fine
Not for me.

**Comment by WereDev** [comment]
I am having a similar issue.  In fact, none of the Alt+ key combinations work for opening a menu.  For instance, Alt+F should open the file menu, but it does nothing on my machine.  I have no idea why this is an issue as it works in my other applications.

Win10, VSC 1.14.2

**Comment by JackTrapper** [comment]
> As mentioned in the atom issue, the underlying issue is related to electron, not vscode.

It may be *because* of electron; but vscode is the one that decided to *use* electron.

If electron is the problem, and it's unsolvable, then its try to move to a different UI framework - one based on native controls.

**Comment by jasenf** [comment]
I swear this was working. 

It's always been my preferred way of saving files.  But now, in the FIle menu, save Workspace has taken over the "S" spot for a shortcut key.  Maybe if we could just change the shortcut keys around?

**Comment by WereDev** [comment]
@JackTrapper @jasenf 

This was updated in another ticket, thought it was here too.  The problem was Scroll Lock.  When it's enabled, the Alt key menu stuff doesn't work right.  Everything is fine when it's disabled.  Not sure if you're experiencing the same thing or if you've come across something different.

**Comment by jasenf** [comment]
I think my issue is new.  The Alt-F then S sequence is now taken by Save Workspace.  Pretty sure until a recent release the S was to save a file.

**Comment by JackTrapper** [comment]
It's something different. 

- Press `Alt`
- Press `F`
- Release `F`
- Press `S`
- Release `S` *(no save)*
- Release `Alt`

The problem is that people don't read the **Steps to Reproduce**, and just scroll through; thinking they get the gist.

**Comment by bpasero** [comment]
We should support this with the new custom menu implementation.

**Comment by sbatten** [comment]
related #52935

**Comment by bpasero** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/52935
